packages/taxdat/tests/testthat/test_read_taxonomy_data_sql.R

```diff
@@ -0,0 +1,68 @@
+username=Sys.getenv("username")
+password=Sys.getenv("password")
+
+library(testthat)
+#test read_taxonomy_data_sql function
+test_that("read_taxonomy_data_sql works",{
+  expect_error(
+    taxonomy_data=read_taxonomy_data_sql(username=NULL,password=password,locations=14)
+    )
+  expect_error(
+    taxonomy_data=read_taxonomy_data_sql(username=username,password=NULL,locations=14)
+    )
+  expect_error(
+    taxonomy_data=read_taxonomy_data_sql(username=username,password=password,locations=NULL)
+    )
+  expect_error(
```

Comment:
I'm not sure what this test is saying about the functions exactly?

Reply:
This test is used for the situation where if someone wants to pull data for a specific oc and forgets to specify the location name, then it'll stop here since location names can't be NULL.

Reply:
there's no OC specified?

Reply:
ah yeah, right, will update with the uid of OC.